apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsProseControl.tsx

```diff
@@ -1,16 +1,16 @@
 import type { ControlProps, RankedTester } from "@jsonforms/core"
 import { Box, FormControl } from "@chakra-ui/react"
-import { hasType, rankWith, schemaMatches } from "@jsonforms/core"
+import { rankWith } from "@jsonforms/core"
 import { withJsonFormsControlProps } from "@jsonforms/react"
 import { FormLabel, Textarea } from "@opengovsg/design-system-react"
 
 import { JSON_FORMS_RANKING } from "~/constants/formBuilder"
 
 export const jsonFormsProseControlTester: RankedTester = rankWith(
   JSON_FORMS_RANKING.ProseControl,
-  schemaMatches(
-    (schema) => hasType(schema, "array") && schema.format === "prose",
-  ),
+  (_, schema) => {
+    return schema.format === "prose"
+  },
```

Comment:
@dcshzj should i also be using `schemaMatches` here?

Reply:
Hmm `schemaMatches` should be correct, this should be referring to the underlying JSON schema and not the uischema. Is the `format: prose` key added to the Prose component schema?

Reply:
yea it is

Reply:
i'll add back `schemaMatches`